interview-app/src/main/java/ua/edu/ukma/interviewapp/InterviewApplication.java

```diff
@@ -0,0 +1,13 @@
+package ua.edu.ukma.interviewapp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class InterviewApplication {
+
+    public static void main(String[] args) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.FinalParametersCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter args should be final.
<!-- __reviewdog__:ChBjNzQ1NzcxMmQ1M2UwYTkwEgpjaGVja3N0eWxl -->